org-gen/members.yaml

```diff
@@ -0,0 +1,233 @@
+members:
```

Comment:
Members.yaml: people who are part of istio org but not in developers or maintainers?  Members can request to be in developers on demand??

Developers: people who are in the developers group, can issue ok to test and bot commands to retest things?

Maintainers: maintainers of each WG.

Reply:
I split this up so members and developers are in a different file and added some more comments on this.

Reply:
that is cool, so no duplicate ids among the member.yaml and developer.yaml.